### PR TITLE
Remove all #if 0...#endif and some other dead code

### PR DESCRIPTION
**Pull request by alerque**
This should be a complete no-op code wise. Nothing in this commit should
change the function of the program in any way. If `git bisect` or
similar ever traces a problem to this commit, then something is wrong
and I didn't match start/end markers properly.

This partially addresses issue #894, bullet point 1 & part of 2.

### CODE REVIEW

**Comment by alerque** [comment]
I just forced-pushed an amended commit to poke the CI builds which were not working upstream at the time I submitted this. Exactly 0 code was changed between commits, and this still merges cleanly to master.

**Comment by alerque** [comment]
@greg-hellings Continuing conversation from #965 here (see my objection to the differences in the duplicate PR).

The [last expression I saw](https://github.com/crosswire/xiphos/issues/894#issuecomment-385499419) from @karlkleinpaste suggested that he might be warming up to this idea.

> perhaps it's time to make them go away.

File: src/main/display.cc [hunk 1]
```diff
@@ -1184,28 +1181,7 @@ GTKChapDisp::getVerseBefore(SWModule &imodule)
...
-		buf = g_strdup_printf("%s<br />", cVerse.GetText());
-#else
-# if 0
-		buf = g_strdup_printf("%s<br />",
-				      (strongs_or_morph
-					   ? block_render(imodule.renderText().c_str())
-					   : imodule.renderText().c_str()));
-# else
-		// gosh this is gross.
+		// begin gosh this is gross.
```
Comment (yetist) [inline src/main/display.cc]: Why you do this unrelated changes?

File: src/main/display.cc [hunk 2]
```diff
@@ -1239,8 +1215,9 @@ GTKChapDisp::getVerseBefore(SWModule &imodule)
 		buf = g_strdup_printf("%s<br />",
 				      (strongs_or_morph ? block_render(divBuf->str) : divBuf->str));
 		g_string_free(divBuf, TRUE);
-# endif
-#endif /* !0 */
+
+    // end grossness
+
```
Comment (yetist) [inline src/main/display.cc]: Why you do the unrelated changes?

File: src/main/display.cc (hunk 1 above)
Comment (alerque) [reply src/main/display.cc]: It isn't unrelated. Without the wrapping `#if 0 ... else ... end` block that got removed, it was no longer clear what section of code the remaining comment referred to. I defined the region to not loose the little bit of information provided by the fact that this code was previously grouped together.

File: src/main/display.cc (hunk 2 above)
Comment (alerque) [reply src/main/display.cc]: Not unrelated. See [this explanation](https://github.com/crosswire/xiphos/pull/913#discussion_r360250752).

**Comment by alerque** [comment]
Ping. @greg-hellings or whoever, do we have any progress on this? Every time I open up the code to review something I bump into these dead blocks and get frustrated. It's really a lot easier on contributors if dead code is left in git history instead of in the files (and if PRs get reviewed and merged within at least a few years).

**Comment by greg-hellings** [comment]
@alerque - I'm perfectly comfortable with this being merged, but when I had made the same proposal many years ago @karlkleinpaste  had expressed his desire to keep this code in place rather than in git history. I would rather not hit the Merge button over his wishes without hearing from him directly that it's OK. If he wants this merged, all he needs to do is click the "Merge pull request" button, so if he hasn't then I don't feel right going over his choice. He's the project lead.

**Comment by karlkleinpaste** [comment]
i'll just mention that i'm about to merge this, but FYI i'm going to be doing a little more beyond this to get rid of the "gosh this is gross" section specifically because of more recent understanding about self-closing div and get rid of the gross hack entirely.

as for keeping around old #if 0 blocks... in my defense, it was a habit learned of much brutal experience literally decades in the past, and prior to the advances of things like git. it no longer matters as much to me as it once did.

**Comment by alerque** [comment]
Thanks @karlkleinpaste. I cut my teeth writing code before versions control too. I have no idea how I survived those dreadful years.
